EscelForce.Business.Login/Services/MapExtraction/CreateExtractionMapService.cs

```diff
@@ -65,18 +66,22 @@ public ServiceResponseModel<bool> SubmitOnObjectSelection(string objectName)
                     Objects = new List<SfObject>()
                 };
 
-                if (!query.Objects.Any(x => x.Name == objectName))
+                string selectedObjectName = SfObject.GetApiNameFromDisplayName(objectName);
+                string label = SfObject.GetObjectNameFromDisplayName(objectName);
```

Comment:
No need to create a separate variable for `label`. This can be directly assigned to line 80.

Reply:
Done. Please check